auv_navigation/auv_mapping/scripts/gate_trajectory_publisher.py

```diff
@@ -29,7 +29,8 @@ def __init__(self):
         self.world_frame = "odom"
 
         self.selected_frame = "gate_blue_arrow_link"
-
+        self.set_enable_service = rospy.Service('taluy/set_gate_trajectory_enable', SetBool, self.handle_enable_service)
```

Comment:
remove taluy here, this node will be placed in `<group ns="$(arg namespace)">` already

Also, rename the service to something like `set_transform_gate_trajectory`?

Reply:
also at the beggining there is a line:

```py

 self.set_object_transform_service = rospy.ServiceProxy(
            "/taluy/map/set_object_transform", SetObjectTransform
        )
```

this must be `map/set_object_transform` due to the similar reasons.

Reply:
it would be even better if these can be obtained from ros params as well


```py
 self.frame1 = rospy.get_param("~gate_frame1", "gate_blue_arrow_link")
 self.frame2 = rospy.get_param("~gate_frame2", "gate_red_arrow_link")
 self.new_frame_away = rospy.get_param("~gate_exit_frame", "gate_exit")
 self.new_frame_closer = rospy.get_param("~gate_enterance_frame", "gate_enterance")
 self.selected_frame = rospy.get_param("~gate_selected_frame", "gate_blue_arrow_link")
 self.world_frame = rospy.get_param("~world_frame", "odom")
```

of course, launch files needs to be adjusted as well